### DISCUSSION TITLE
Danalock V3-BTZB/V3-BTZBE

### DISCUSSION THREAD

**Discussion by sim186** [Koenkk/zigbee2mqtt#12725] [Devices]
As someone problem to get status updates from this lock in home assistant?

Looks like when the lock restart it works for a while but then stop suddenly to communicate the status. Should I setup some special settings?

**Comment by portboy** [comment]
I've problems like this with this device too.
It only works for a while .... and then it stops until I restart the docker container.
The time frame it works is variable, somtimes for a week, sometimes only for minutes.

**Reply by portboy** [reply]
No hint to solve this problem?
How can I investigate to solve it by myself ?

**Reply by sim186** [reply]
For me is still like that. Maybe we can try to write to the vendor?

**Reply by portboy** [reply]
I made a factory reset and set up from scratch the  the connection to zigbee2nnqtt. Since one week it's stable.

**Reply by teras** [reply]
How's the connection with HA? I am planning to buy this device since it is the only one supported for Europe (if I am not wrong) and being a zigbee (not z-wave) device

**Reply by portboy** [reply]
**Don't buy it**. It ist still unstable. 

I have to restart zigbee2mqtt every week. But sometime I have to re-pair the device.

**Reply by portboy** [reply]
The status update works only for some minutes after re-pair.
